pkg/client/unversioned/remotecommand/remotecommand.go

```diff
@@ -194,12 +194,19 @@ func (e *Streamer) doStream() error {
 		if err != nil {
 			return err
 		}
-		defer remoteStdin.Reset()
```

Comment:
don't you still need the deferred remoteStdin.Reset() in the case where the remoteStdin `cp` never completes, or does the conn.Close() cover that?